src/components/change-view.jsx

```diff
@@ -111,9 +111,9 @@ export default class ChangeView extends React.Component {
         }
 
         return (
-            <div className="change-view">
-                {this.renderVersionSelector(page)}
+            <div className="change-view-container">
```

Comment:
This was an artifact from way before when I initially branched. I changed it back to reflect convention and also got rid of unnecessary styles associated with it.